test cases/nasm/3 nasm only/meson.build

```diff
@@ -0,0 +1,13 @@
+project('nasm only')
+
+if not add_languages('nasm', required: false)
+  error('MESON_SKIP_TEST: nasm not found')
+endif
+
+sources = files('dummy.asm')
+
+dummy = library(
+    'dummy',
+    sources,
+    vs_module_defs: 'dummy.def',
+)
```

Comment:
Better merge this unit test in the existing one, creating small tests like this slow down the CI.

Reply:
All the existing tests merge NASM with C, thus implicitly working around the bug I'm trying to fix.

Reply:
Oh, I forgot ot mention that the existing tests are Linux only by virtue of using their syscall convention (int 80h). I've got one stashed for Windows, let me know if you'd like it.